scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java

```diff
@@ -191,6 +202,22 @@ protected String getJobStatusFieldColor(ListGridRecord record, int rowNum, int c
         return base;
     }
 
+    private boolean selectJobIdFromUrl() {
+        String urlJobId = Window.Location.getParameter("job");
+        if (urlJobId == null) {
+            return true;
+        }
```

Comment:
you can replace this if by: return (urlJobId != null)

Reply:
Are you sure? The rest of the function won't be executed

Reply:
yes you are right. Sorry